OracleDatabase/SingleInstance/extensions/sharding/README.md

```diff
@@ -0,0 +1,7 @@
+# Sharding Extension
+Sharding extension is required to build the catalog and shard containers. When
+the SingleInstance container image is extended with **sharding** extension, it downloads the 
+required scripts from [db-sharding/docker-based-sharding-deployment](https://github.com/oracle/db-sharding/tree/master/docker-based-sharding-deployment)
+repository, and package them with the SingleInstance container image to form extended image.
```

Comment:
-> "packages"

-> "to form an extended image" (or "the extended")

Reply:
Done